portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+        <h1>My Gallery</h1>
+        <p>Photography is the way I capture moments of the world around me. Photos allow me to remmeber important moments and people by
+         providing me with a reminder of them. Most of my photos fall into one of three categories. </p>
+        <div id="first">
```

Comment:
Perhaps these div ids could have more meaningful names, corresponding to their content?

Reply:
I've changed the names and while some (the three ending in "Third")may still seem to be generic, I'm naming them such in advance of my adding CSS with the layout I have planned.